src/sentry/api/serializers/rest_framework/base.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+import six
+from django.utils.text import re_camel_case
+from rest_framework.fields import empty
+from rest_framework.serializers import ModelSerializer
+
+
+def camel_to_snake_case(value):
+    """
+    Splits CamelCase and converts to lower case with underscores.
+    """
+    return re_camel_case.sub(r'_\1', value).strip('_').lower()
+
+
+def snake_to_camel_case(value):
+    """
+    Converts a string from snake_case to camelCase
+    """
+    words = value.strip('_').split('_')
+    return words[0].lower() + ''.join([word.capitalize() for word in words[1:]])
+
+
+def convert_dict_case(obj, converter):
```

Comment:
```suggestion
def convert_dict_key_case(obj, converter):
```
Slightly clearer. But also :bike: